bmorph/core/models.py

```diff
@@ -267,7 +291,7 @@ def create_lstm(train_shape: tuple, depth: int=1,
     from tensorflow.keras.models import Sequential
     from tensorflow.keras.layers import Dense
     from tensorflow.keras.layers import LSTM
-    from keras.constraints import NonNeg
+    from tensorflow.keras.constraints import NonNeg
```

Comment:
Is this a new syntax in tensorflow 2?

Reply:
This is to ensure that we are using `tensorflow` explicitly. Keras also is an interface for `theano`, so we want to make sure we don't import that instead.